bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/Tr064Comm.java

```diff
@@ -494,7 +497,7 @@ private SOAPMessage readSoapResponse(String soapActionHeader, SOAPMessage reques
             // Make sure connection is released. If error occurred make sure to print in log
             if (exceptionOccurred) {
                 logger.error("Releasing connection to FritzBox because of error!");
```

Comment:
Please convert this to a WARN instead of ERROR.

Reply:
:+1: